go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/BTrDB/btrdb/v5
+
+go 1.14
+
+require (
+	github.com/BTrDB/btrdb v5.9.1+incompatible
+	github.com/golang/protobuf v1.4.0
+	github.com/grpc-ecosystem/grpc-gateway v1.14.4
+	github.com/huichen/murmur v0.0.0-20130808212358-e0489551cf51
+	github.com/op/go-logging v0.0.0-20160315200505-970db520ece7
+	github.com/pborman/uuid v1.2.0
+	golang.org/x/net v0.0.0-20200501053045-e0ff5e5a1de5
+	google.golang.org/genproto v0.0.0-20200430143042-b979b6f78d84
+	google.golang.org/grpc v1.29.1
```

Comment:
Can you change this to 1.23.1 ? There's a bug in 1.29.1

Reply:
done